ui/src/app/cohort-search/cohort-criteria-menu.spec.tsx

```diff
@@ -33,7 +33,6 @@ describe('CohortCriteriaMenu', () => {
     serverConfigStore.set({
       config: {
         ...defaultServerConfig,
-        enableDrugWildcardSearch: false,
       },
     });
```

Comment:
We should remove the entire `serverConfigStore` reference since it was only added to the test for the `enableDrugWildcardSearch` flag.

Reply:
Thank you for clarifying!